js/ui/discoveryFeedButton.js

```diff
@@ -4,6 +4,7 @@ const { Clutter, Gio, GLib, GObject, St } = imports.gi;
 
 const Lang = imports.lang;
 const Main = imports.ui.main;
+const Clubhouse = imports.ui.components.clubhouse;
```

Comment:
I understood from the Slack discsussion that this commit isn't needed anymore?

Reply:
Yes, I'll remove this commit to don't touch the discovery feed.